.github/workflows/workflow.yml

```diff
@@ -3,9 +3,6 @@ name: Scully build, test, package, and deploy
 on:
   # Trigger the workflow on push or pull request,
   # but only for the main branch
-  push:
-    branches:
-      - main
```

Comment:
Should this not be added again?

Reply:
if removal is intentional, I think it's best to adjust comment line above as well

Reply:
The change is intentional. I agree about the comment, I'll update it.
A push to main is _only_ happening after a merge. A pr will only be merged after all tests are passed.
So, the previous setup would run all the tests again _after_ the merge.
Also, it started the tests after the version bump did its work, and I don't think we need/want to test version bumps.